tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/distribution/ClusterTestBase.java

```diff
@@ -751,6 +762,10 @@ protected void sendInRange(final int node,
                message.putStringProperty(ClusterTestBase.FILTER_PROP, new SimpleString(filterVal));
             }
 
+            if (routingType != null) {
```

Comment:
this bit still seems unnecessary, or am I missing something?

Reply:
I don't see the need for any change to the clustertestbase

Reply:
I think those changes are needed, otherwise send defaults to MULTICAST. Since I am sending messages to a broker without a consumer in the test then those messages would otherwise get discarded... right? I can have another look at it if you think otherwise.

Reply:
that may be some special case that requires a sent message to have a routing semantic, it is not normal. Typically a message goes to an address, and the address has a routing type configured across bindings. Ie: it is the broker that determines how a message is routed.
It is non typical to call setRoutingType - it should not be necessary. From a quick peek at git - ARTEMIS-1068 introduced that setter for AMQP, probably for some amqp to jms mapping scenario. But it should not be used or required in this scenario.
I think, have another look.